Gzz/animation

JG-e opened the pull request:
Added animation to walking. Other animations are waiting for art team if there are.

JG-e committed:
Animation Test Passed for Walking

JG-e committed:
Added better animation logic

maxthewang reviewed (commented):
The branch works on its own (aside from the walking animation happening constantly), but after a while of trying to merge it and resolve the conflicts in different ways the animation still doesn't work on my end

JG-e committed:
Added gitignore

JG-e committed:
Added Grab to the animation but hasn't yet work

JG-e committed:
Added Grab animation when trying to interact

JG-e committed:
nimation for the main char

maxthewang reviewed (approved):
Walking and grabbing animations work perfectly now. Later on we can also ensure the grabbing animation only happens when an item is discovered

maxthewang committed:
SFX and music (discoverables, win/loss, ambience)
New audio assets implemented according to Dima's directions

maxthewang committed:
Polishing from beta
Larger player collider, build graphics issue, notepad speed, and other visuals

maxthewang committed:
Small crime scene polish
Fixed notepad (right page information wouldn't display for some reason), camera position and collider refinements

JG-e committed:
Add new police animations and updated interrogation and intro scenes

maxthewang committed:
Repositioned some assets, added interrogation camera animation

maxthewang reviewed (approved):
Adjusted it a bit, everything looks ready to merge now if there's nothing else to add

JG-e committed:
Changed the interrogator to the new model.
Added new animation + transitions for the new model

JG-e committed:
Add options action for escape key and update scene properties

JG-e committed:
Added the animations for police and added the new interrogator.

JackyTruong committed:
Added controller support to Questions

AxisShine committed:
Adding voicelines to assets

JG-e committed:
updated main char model and refined gun holding officer position + added interrogator animation manager.

AxisShine committed:
Renamed all the new voicelines accordingly

AxisShine committed:
Updated voiceline asset file

JG-e committed:
Add new idle and handcuff animations for main character; remove outdated handcuff animation

JG-e committed:
Fine tuned animation in interrogation scene.

AxisShine committed:
Added blood next to gun and fire blocking door

AxisShine committed:
Merge pull request #54 from maxthewang/Beta-updates-Alice

Added blood next to gun and fire blocking door

JG-e committed:
Relocated resource files and added voiceover for dialogs

maxthewang committed:
Reduced flame screen flashing
Not gone, but more reasonable and fitting now

AxisShine committed:
Added voicelines to interactables, and their updated clues.

maxthewang committed:
Turning around animation at end of crime scene

AxisShine committed:
Added proper keypad

JG-e committed:
Add StopPlayingVoicelines method to DialogManager for voice control

AxisShine committed:
Added boxes to garage

maxthewang committed:
More animations (breathing, zoom out)

JG-e committed:
Merge pull request #57 from maxthewang/gzz/introVoice

Gzz/intro voice

AxisShine committed:
Moved a couple items around for better visibility and story telling

JG-e committed:
Restructured the audio files

JG-e committed:
Hot fix: Text went wrong, the audio source missed from rebase

maxthewang committed:
Stopped movement during gun animation, updated avatars

JG-e committed:
Temp

JG-e committed:
Added SFX for the reaction + grab sound, not working for some reason.

JackyTruong committed:
fixed interrogation sfx bug

fixed an interrogation sfx bug where the clip of their response was not being played.

JG-e committed:
Moved SFX

JG-e committed:
relocated/renamed audio files

JG-e committed:
Done with interrogation sound

JackyTruong committed:
Merge pull request #63 from maxthewang/gzz/interrogationAnimeAndSFX

Gzz/interrogation anime and sfx into animation
